determined-ai/determined
# docs: add documentation for setting up Slack Webhooks [WEB-215]

## Description
docs: add documentation for setting up Slack Webhooks [WEB-215]

This PR adds the `Through Slack` section referenced in https://github.com/determined-ai/determined/pull/5244

Adds documentation for setting up Slack Webhooks in Determined. 

Screenshot:

![7F88E296-7233-43AD-8D36-C9903AAB6B9C](https://user-images.githubusercontent.com/103522725/196290898-cbba9447-2c7e-49f0-9f8c-f697bccbda51.jpeg)




## Test Plan

1. Run `python /docs/live.py` to build the documentation locally. 
2. Navigate to `/html/integrations/notification/slack.html`
3. Read through documentation and ensure everything is clear and formatted correctly.




## Commentary (optional)





[WEB-215]: https://determinedai.atlassian.net/browse/WEB-215?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
